thrust/thrust/system/cuda/detail/transform_scan.h

```diff
@@ -28,6 +28,8 @@
 
 #include <thrust/detail/config.h>
 
+#include "thrust/iterator/transform_iterator.h"
+
```

Comment:
On missed include on the wrong spot

Reply:
Now I have fallen as well to auto-included includes :(